proposals/low-level-struct-improvements.md

```diff
@@ -756,17 +756,7 @@ The runtime repository has several non-public APIs that capture `ref` parameters
 
 The majority of these APIs will likely choose to have proper lifetime tracking on the return which will be achieved simply by updating to C# 11. A few though will want to keep their current semantics of not tracking the return value because their entire intent is to be unsafe. The most notable examples are `MemoryMarshal.CreateSpan` and `MemoryMarshal.CreateReadOnlySpan`. This will be achieved by marking the parameters as `scoped`.
 
-That means the runtime needs an established pattern for unsafely removing `scoped` from a parameter. This can be done today via a combination of existing methods:
-
-```c#
-Span<T> CreateSpan<T>(scoped ref T value, int length)
-{
-    ref T local = Unsafe.AsRef<T>(Unsafe.AsPointer(ref value));
-    return new Span<T>(local, length);
-}
-```
-
-This will work but is likely going to result in unnecessary code generation. One other consideration is that either:
```

Comment:
I would argue that the fact one can bounce through a pointer to remove `scoped` is, in fact, an important consideration.  In the absence of proper support from the language for removing `scoped`, developers will end up with exactly this code.

This note is incorrect, however, as it will definitely not work in some cases because it's a potential GC hole.  The fact that this is difficult for most developers to reason about makes it a serious pit of failure with the current `scoped ref` feature.